src/components/TopNav.js

```diff
@@ -126,43 +126,43 @@ class TopNav extends Component {
               </Box>
```

Comment:
This diff looks worse than it is. I just cut and pasted a Grid item and moved it up.